iit/model_pairs/strict_iit_model_pair.py

```diff
@@ -68,9 +97,9 @@ def run_train_step(
     ) -> dict:
         use_single_loss = self.training_args["use_single_loss"]
 
-        iit_loss = 0
-        ll_loss = 0
-        behavior_loss = 0
+        iit_loss = t.zeros(1)
```

Comment:
I'm not completely sure why this is needed- You can usually add floats and tensors without messing up the grads, right?

Reply:
This is for mypy type-checking. step_on_loss expects a Tensor instead of a float and the .item() call at the end is a type error if this isn't a Tensor.

I think the right way to resolve this is to remove the if isintance(Tensor) logic at the end of the function, since it's now always a tensor. I'll do that.